client/components/themes-list/docs/example.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import noop from 'lodash/noop';
+
+/**
+ * Internal dependencies
+ */
+import ThemesList from 'components/themes-list';
+
+const onScreenshotClickFn = function( theme, index ) {
+	console.log( `Theme ${ theme.id } at ${ index } clicked` );
+};
+
+const demoThemes = [ {
+	id: 'twentyfourteen',
+	name: 'Twenty Fourteen',
+	screenshot: '//i1.wp.com/theme.wordpress.com/wp-content/themes/pub/twentyfourteen/screenshot.png',
+	actionLabel: 'Click Action Theme 1',
+	onScreenshotClick: onScreenshotClickFn,
+},
+{	id: 'twentyfifteen',
+	name: 'Twenty Fifteen',
+	screenshot: '//i1.wp.com/theme.wordpress.com/wp-content/themes/pub/twentyfifteen/screenshot.png',
+	actionLabel: 'Click Action Theme 2',
+	onScreenshotClick: onScreenshotClickFn,
+},
+{	id: 'twentysixteen',
+	name: 'Twenty Sixteen',
+	screenshot: '//i1.wp.com/theme.wordpress.com/wp-content/themes/pub/twentysixteen/screenshot.png',
+	actionLabel: 'Click Action Theme 3',
+	onScreenshotClick: onScreenshotClickFn,
+} ];
+
+export default React.createClass( {
+	displayName: 'ThemesListExample',
+
+	getActionLabel: theme => theme.actionLabel,
+
+	getButtonOptions() {
+		return {
+			action1: {
+				label: 'Menu Item 1',
+				action: function() {
+					console.log( 'Menu Item 1 selected' );
+				}
+			},
+			action2: {
+				label: 'Menu Item 2',
+				action: function() {
+					console.log( 'Menu Item 2 selected' );
+				}
+			}
+		}
+	},
+
+	onScreenshotClick: ( theme, index ) => theme.onScreenshotClick( theme, index ),
```

Comment:
This, along with `getActionLabel`, should read:

```js
onScreenshotClick( theme, index ) {
  theme.onScreenshotClick( theme, index );
}
```

That is, use class method shorthand. However, now looking at the implementation, this is pretty redundant. I'd either bind each theme's `onScreenshotClick` to the theme itself upon instantiation of `demoThemes`, _or_ I'd skip copying `onScreenshotClickFn` into each theme object and work with `onScreenshotClickFn` directly in this method here.